pygromos/analysis/error_estimate.py

```diff
@@ -0,0 +1,106 @@
+"""
+File: calculation of error estimates
+Warnings: this class is WIP!
+
+Description:
+    Implementation of functions estimating the Error for calculated values using Gromos
+
+Author: Paul Katzberger
+"""
+
+from typing import List
+import numpy as np
+
+class ee:
```

Comment:
Hm, maybe we could name it error_estimate? (descriptive variables make reading easier/ we are not any more in the age, when chars were short ;) )

Reply:
maybe error_estimator

Reply:
Yes sounds good, I will do the refactoring